integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java

```diff
@@ -7,15 +7,13 @@
 
   import org.json.JSONObject;
   import org.junit.Test;
-  import org.opensearch.sql.legacy.SQLIntegTestCase;
-
+  
   import java.io.IOException;
 
   import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_DATE;
   import static org.opensearch.sql.util.MatcherUtils.rows;
   import static org.opensearch.sql.util.MatcherUtils.schema;
-  import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
-  import static org.opensearch.sql.util.MatcherUtils.verifySchema;
+    import static org.opensearch.sql.util.MatcherUtils.verifySchema;
```

Comment:
```suggestion
import static org.opensearch.sql.util.MatcherUtils.verifySchema;
```

Reply:
Addressed in 65df31b37e9b2766d0a5fd4c31745fdc3638d4af